mappings/net/minecraft/entity/LivingEntity.mapping

```diff
@@ -17,6 +17,9 @@ CLASS ahu net/minecraft/entity/LivingEntity
 	FIELD bG lastAttackTime I
 	FIELD bJ absorptionAmount F
 	FIELD b LOGGER_LIVING Lorg/apache/logging/log4j/Logger;
+	FIELD bf velocityWalk F
+	FIELD bg velocityVert F
```

Comment:
Vert? Do we need the shortcut?

Reply:
hmm, I'll see how it feels with `velocityVertical`.

Reply:
These should be using suffix naming.

`walkVelocity`
`verticalVelocity`

Reply:
And shouldnt `walk` just be `horizontal`?

Reply:
> These should be using suffix naming.

I'm going off the scheme of `velocityX`, `velocityY`, and `velocityZ` here. We might wanna change those too, though. Should we?

> And shouldnt `walk` just be `horizontal`?

Well, strafing is horizontal too, and arguably *more* horizontal than walking forward. I'm using the names from Controls for these.